woocommerce/woocommerce
# Migrating product editor inventory section to use slot fills

### All Submissions:






### Changes proposed in this Pull Request:



Transitioning inventory section in product editor to utilize our slot-fill components. 

**Note:** There were some preexisting tests for the Inventory section. I experimented for a bit with making these work via slot-fill, but it didn't seem to be a simple thing to make work, and I don't see any other instances in the repo of testing slot-filled content. This may need to be a spike or another issue.

Closes #36420 




### How to test the changes in this Pull Request:



1. Checkout branch.
2. Enable product editor at WooCommerce -> Settings -> Advanced -> Features -> Try the new product editor
3. Go to product editor at Products -> Add new (MVP)
4. To general smoke test of inventory section, ensure it behaves exactly the same as previously.



### Other information:




### FOR PR REVIEWER ONLY:
